examples/api/filled_step.py

```diff
@@ -172,7 +172,7 @@ def stack_hist(ax, stacked_data, sty_cycle, bottoms=None,
 hist_func = partial(np.histogram, bins=edges)
 
 # set up style cycles
-color_cycle = cycler('facecolor', 'rgbm')
+color_cycle = cycler('facecolor', plt.rcParams['axes.prop_cycle'][:4])
```

Comment:
With a new cycler release, we can also fix the key normalization problem
for property cycles via change_key()

On Sun, Feb 14, 2016 at 3:01 PM, Thomas A Caswell <notifications@github.com>
wrote:

> In examples/api/filled_step.py
> <https://github.com/matplotlib/matplotlib/pull/5997#discussion_r52847857>:
>
> > @@ -172,7 +172,7 @@ def stack_hist(ax, stacked_data, sty_cycle, bottoms=None,
> >  hist_func = partial(np.histogram, bins=edges)
> >
> >  # set up style cycles
> > -color_cycle = cycler('facecolor', 'rgbm')
> > +color_cycle = cycler('facecolor', plt.rcParams['axes.prop_cycle'][:4])
>
> I am currently working on sorting out what needs to be done for a cycler
> release, hopefully I can get this done today [image: :shipit:] .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/5997/files#r52847857>.
>